src/backend/bs_repserver

```diff
@@ -1010,6 +1010,7 @@ sub getbinarylist {
     @bins = grep {!/^::import::/} @bins if $cgi->{'noimport'};
     @bins = BSRepServer::Containertar::add_containers(@bins) if grep {/\.containerinfo$/} @bins;
     @bins = filtersources(@bins) if $cgi->{'nosource'} || -e "$reporoot/$prp/$arch/$packid/.nosourceaccess";
+    @bins = grep {!/^_ccache.tar.gz$/} @bins unless $cgi->{'withccache'};
```

Comment:
Hmm, the `.` in the regexp seem to need escaping.

Come to think of it: why do you use a regexp at all in this case? It could be simply:
```
@bins = grep {$_ ne '_ccache.tar.gz'} @bins unless $cgi->{'withccache'};
```